src/software/Localization/main_cameraLocalizer.cpp

```diff
@@ -270,6 +273,9 @@ int main(int argc, char** argv)
           "an optimal value.")
       ("useFrameBufferMatching", po::bool_switch(&useFrameBufferMatching), 
           "[voctree] Enable/Disable the matching with the last N frame of the sequence")
+      ("noRobustMatching", po::bool_switch(&noRobustMatching), 
```

Comment:
it's better to avoid negative options: "switch on noRobustMatching to disable robustMatching" ;)